zest/releaser/bzr.py

```diff
@@ -24,10 +24,10 @@ def name(self):
         return dir_name
 
     def available_tags(self):
-        tag_info = execute_command('bzr tags')
-        tags = [line[:line.find(' ')] for line in tag_info.split('\n')]
+        tag_info = execute_command([u'bzr', u'tags'])
+        tags = [line[:line.find(u' ')] for line in tag_info.split(u'\n')]
         tags = [tag for tag in tags if tag]
-        logger.debug("Available tags: %r", tags)
+        logger.debug(u"Available tags: {0!r}".format(tags))
```

Comment:
You'll see a lot of this in this pull request. Mostly added to make consistent use of PEP-3101 rather than a mixture of that and string interpolation, however this is just a style preference and again I'm happy to change it back for logging.